src/rules-by-scope.ts

```diff
@@ -365,124 +365,124 @@ const reactPerfRules = {
-  'unicorn/no-useless-undefined': 'off',
+  'unicorn/no-useless-promise-resolve-reject': 'off',
   'unicorn/no-zero-fractions': 'off',
   'unicorn/number-literal-case': 'off',
   'unicorn/numeric-separators-style': 'off',
-  'unicorn/prefer-add-event-listener': 'off',
+  'unicorn/no-useless-undefined': 'off',
   'unicorn/prefer-array-flat': 'off',
   'unicorn/prefer-array-flat-map': 'off',
   'unicorn/prefer-array-some': 'off',
+  'unicorn/prefer-add-event-listener': 'off',
   'unicorn/prefer-blob-reading-methods': 'off',
   'unicorn/prefer-code-point': 'off',
   'unicorn/prefer-date-now': 'off',
-  'unicorn/prefer-dom-node-append': 'off',
   'unicorn/prefer-dom-node-dataset': 'off',
   'unicorn/prefer-dom-node-remove': 'off',
+  'unicorn/prefer-dom-node-append': 'off',
   'unicorn/prefer-dom-node-text-content': 'off',
   'unicorn/prefer-event-target': 'off',
   'unicorn/prefer-includes': 'off',
-  'unicorn/prefer-logical-operator-over-ternary': 'off',
   'unicorn/prefer-math-min-max': 'off',
-  'unicorn/prefer-math-trunc': 'off',
+  'unicorn/prefer-logical-operator-over-ternary': 'off',
   'unicorn/prefer-modern-dom-apis': 'off',
-  'unicorn/prefer-modern-math-apis': 'off',
-  'unicorn/prefer-native-coercion-functions': 'off',
+  'unicorn/prefer-math-trunc': 'off',
   'unicorn/prefer-node-protocol': 'off',
+  'unicorn/prefer-modern-math-apis': 'off',
   'unicorn/prefer-number-properties': 'off',
   'unicorn/prefer-optional-catch-binding': 'off',
+  'unicorn/prefer-native-coercion-functions': 'off',
   'unicorn/prefer-prototype-methods': 'off',
   'unicorn/prefer-query-selector': 'off',
   'unicorn/prefer-reflect-apply': 'off',
-  'unicorn/prefer-regexp-test': 'off',
   'unicorn/prefer-set-size': 'off',
+  'unicorn/prefer-regexp-test': 'off',
   'unicorn/prefer-spread': 'off',
   'unicorn/prefer-string-replace-all': 'off',
   'unicorn/prefer-string-slice': 'off',
-  'unicorn/prefer-string-starts-ends-with': 'off',
   'unicorn/prefer-string-trim-start-end': 'off',
+  'unicorn/prefer-string-starts-ends-with': 'off',
   'unicorn/prefer-structured-clone': 'off',
   'unicorn/prefer-type-error': 'off',
   'unicorn/require-array-join-separator': 'off',
-  'unicorn/require-number-to-fixed-digits-argument': 'off',
   'unicorn/switch-case-braces': 'off',
+  'unicorn/require-number-to-fixed-digits-argument': 'off',
   'unicorn/text-encoding-identifier-case': 'off',
   'unicorn/throw-new-error': 'off',
+  'unicorn/no-useless-spread': 'off',
 } as const;
 
 const vitestRules = {
   'vitest/no-conditional-tests': 'off',
   'vitest/no-import-node-test': 'off',
-  'vitest/prefer-each': 'off',
   'vitest/prefer-to-be-falsy': 'off',
+  'vitest/prefer-each': 'off',
   'vitest/prefer-to-be-object': 'off',
-  'vitest/prefer-to-be-truthy': 'off',
   'vitest/require-local-test-context-for-concurrent-snapshots': 'off',
+  'vitest/prefer-to-be-truthy': 'off',
+} as const;
+
+const treeShakingRules = {
+  'tree-shaking/no-side-effects-in-initialization': 'off',
```

Comment:
I do not how now this appears. I need recheck the intergration test where `nusery` is disabled